engines/macventure/stringtable.h

```diff
@@ -89,9 +90,9 @@ class StringTable {
 			char *str = new char[strLength + 1];
 			res->read(str, strLength);
 			str[strLength] = '\0';
-			// HACK until a proper special char implementation is found, this will have to do.
-			Common::String result = Common::String(str);
-			toASCII(result);
+
+			Common::String result = Common::convertToU32String(str, Common::kMacRoman);
```

Comment:
Shouldn't that be of type Common::U32String?

Reply:
fixed!